src/models/subject.ts

```diff
@@ -173,4 +167,33 @@ export class SubjectWrapper {
       ...classificationColumns,
     };
   }
+
+  private addVerification(model: Verification): void {
+    // when a verification decision is made, it usually doesn't have a tag
```

Comment:
terminology: what do you mean it _usually_ doesn't have a tag? in what case does it have a tag?

Reply:
good catch, "usually" is not correct

It will never have the `tag`